### DISCUSSION TITLE
some url return 403

### DISCUSSION THREAD

**Discussion by andyoulovexy** [bunkerity/bunkerweb#404] [Help]
![image](https://user-images.githubusercontent.com/38940994/207343409-63811aa8-fad5-4385-9558-69bf2660f522.png)
Like this:
![image](https://user-images.githubusercontent.com/38940994/207343828-bc33428d-6eba-4555-a51b-176b3e2c4eb6.png)

How to solve it?tks

**Comment by TheophileDiot** [comment]
Hi, looks like the plugin crowdsec denied your requests

**Comment by andyoulovexy** [comment]
> Hi, looks like the plugin crowdsec denied your requests

How to solve it?tks

**Comment by TheophileDiot** [comment]
@andyoulovexy It can be a false positive with Crowdsec so I'd say that you can deactivate the plugin in your configuration and try opening an issue related to this to Crowdsec, maybe they can help you with this since it's related to their tool.

**Comment by andyoulovexy** [comment]
> @andyoulovexy It can be a false positive with Crowdsec so I'd say that you can deactivate the plugin in your configuration and try opening an issue related to this to Crowdsec, maybe they can help you with this since it's related to their tool.

maybe,maybe crowdsec plugin should be fix or update?I will open an issue in crowdsec hub,tks

**Comment by TheophileDiot** [comment]
@andyoulovexy I'll have a check

**Comment by andyoulovexy** [comment]
> @andyoulovexy I'll have a check

![image](https://user-images.githubusercontent.com/38940994/207549104-9090a8a6-4d7a-4ff2-aa3e-24bef072cdb6.png)
I commented these three lines, and there is no problem with revisiting, but I tested based on the nikto tool, and my ip was banned, but revisiting based on this ip can still be accessed normally

**Comment by TheophileDiot** [comment]
@andyoulovexy Your IP was probably banned by the badbehavior because of the number of bad status code you generated

**Comment by andyoulovexy** [comment]
> Your IP was probably banned by the badbehavior because of the number of bad status code you generated

![image](https://user-images.githubusercontent.com/38940994/207550719-0a41f3e8-2347-4a61-b747-9f4bbff2ba32.png)
i didn't turn on bad behavior

**Comment by TheophileDiot** [comment]
@andyoulovexy That's not recommended, except if you're just testing your configuration 😅
Do you have any logs ?

**Comment by TheophileDiot** [comment]
@andyoulovexy It looks like that Crowdsec is still activated so your IP was banned

**Comment by TheophileDiot** [comment]
[bunkerweb.txt](https://paste.techbag.app/?ff1def00833f3166#BhQC6V83eSfKFtpvarsJQBRN4W5oa4tCkmh35sEKEAEx)
[crowdsec_plugin.txt](https://paste.techbag.app/?83e281be74bf59cc#EPqsoqYNuxBMWHt19FPQwJZQ2KqBBVDHv75ZKZN7J8yD)
![image](https://user-images.githubusercontent.com/57175294/207554286-1896d830-ad9f-43b0-9d53-97a51d10917e.png)

@andyoulovexy Your IP address was showing so I deleted your message, here are the files you sent with your IP censored

**Comment by andyoulovexy** [comment]
> [bunkerweb.txt](https://paste.techbag.app/?ff1def00833f3166#BhQC6V83eSfKFtpvarsJQBRN4W5oa4tCkmh35sEKEAEx) [crowdsec_plugin.txt](https://paste.techbag.app/?83e281be74bf59cc#EPqsoqYNuxBMWHt19FPQwJZQ2KqBBVDHv75ZKZN7J8yD) ![image](https://user-images.githubusercontent.com/57175294/207554286-1896d830-ad9f-43b0-9d53-97a51d10917e.png)
> 
> @andyoulovexy Your IP address was showing so I deleted your message, here are the files you sent with your IP censored

[bunkerweb.txt](https://github.com/bunkerity/bunkerweb/files/10226206/bunkerweb.txt)
[crowdsec_plugin.txt](https://github.com/bunkerity/bunkerweb/files/10226208/crowdsec_plugin.txt)
tks！！！

**Comment by andyoulovexy** [comment]
> @andyoulovexy It looks like that Crowdsec is still activated so your IP was banned

![image](https://user-images.githubusercontent.com/38940994/207564912-8721ef37-6f2f-462d-818c-f5ba630615c8.png)
My IP is not banned

**Comment by TheophileDiot** [comment]
> > @andyoulovexy It looks like that Crowdsec is still activated so your IP was banned
> 
> ![image](https://user-images.githubusercontent.com/38940994/207564912-8721ef37-6f2f-462d-818c-f5ba630615c8.png) My IP is not banned

According to your screenshot, you're being reported by Crowdsec and Modsecurity.

**Comment by andyoulovexy** [comment]
> > > @andyoulovexy It looks like that Crowdsec is still activated so your IP was banned
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/38940994/207564912-8721ef37-6f2f-462d-818c-f5ba630615c8.png) My IP is not banned
> 
> According to your screenshot, you're being reported by Crowdsec and Modsecurity.

i will disable modsecurity,i will try it

**Comment by TheophileDiot** [comment]
@andyoulovexy Hmmmm it's okay for now but later it would be better to create rules for modsec.

**Comment by andyoulovexy** [comment]
> 

![image](https://user-images.githubusercontent.com/38940994/207566290-b3e01089-ce0d-487b-b530-ed9a87b7a301.png)
i didn't start modsecurity at all.

**Comment by TheophileDiot** [comment]
@andyoulovexy You just deactivated it or it was already deactivated ?

**Comment by andyoulovexy** [comment]
> @andyoulovexy You just deactivated it or it was already deactivated ?

After adding the service, it is not activated

**Comment by TheophileDiot** [comment]
@andyoulovexy Ok, any changes in the logs then ?

**Comment by andyoulovexy** [comment]
> @andyoulovexy You just deactivated it or it was already deactivated ?

my ip x.x.x.222,it not in crowdsec decision!!!
![image](https://user-images.githubusercontent.com/38940994/207568395-3e111917-7851-48dd-a755-d5ece3f5885c.png)

but in bunkerweb log :
![image](https://user-images.githubusercontent.com/38940994/207568701-9a2b0d32-fd4a-44cf-b08c-2d7fb021a5c7.png)

**Comment by andyoulovexy** [comment]
> @andyoulovexy Ok, any changes in the logs then ?

![image](https://user-images.githubusercontent.com/38940994/207569206-5aacd9c2-fa88-414f-9ac0-e2ac963b9cfb.png)

**Comment by TheophileDiot** [comment]
Ok, it has to be a problem with Crowdsec then, let me know if you have an answer for your issue

**Comment by andyoulovexy** [comment]
> Ok, it has to be a problem with Crowdsec then, let me know if you have an answer for your issue

ok.I think crowdsec-bunkerweb-bouncer had some problem,but i still open an issue in crowdsec.tks

**Comment by andyoulovexy** [comment]
> > Ok, it has to be a problem with Crowdsec then, let me know if you have an answer for your issue
> 
> ok.I think crowdsec-bunkerweb-bouncer had some problem,but i still open an issue in crowdsec.tks
I had add my ip to whitelist,everything is ok!!but i can't add all ip to whitelist!!!!.
![image](https://user-images.githubusercontent.com/38940994/207636529-4eac409b-ab78-43a7-8830-e704c84ebcaf.png)
This logs:
![image](https://user-images.githubusercontent.com/38940994/207637023-3a517fad-92f3-4149-a9c0-bd2345c8e6b3.png)

**Comment by andyoulovexy** [comment]
> Ok, it has to be a problem with Crowdsec then, let me know if you have an answer for your issue

![image](https://user-images.githubusercontent.com/38940994/207757522-9f2551fa-d889-4d8c-8cd7-c6528ddd0402.png)
maybe need check bans.i think exist balcklist cache~.i open an issue at crowdsec discord，crowdsec developer reply crowdsec-bunkerweb-bouncer had some problem！！！https://discord.com/channels/921520481163673640/1003029165425049751/threads/1052533768151384094

**Comment by andyoulovexy** [comment]
> Ok, it has to be a problem with Crowdsec then, let me know if you have an answer for your issue

debug logs:
**2022/12/15 02:57:49 [debug] 1802#1802: *597 [lua] bouncer.lua:376: allowIp(): 'ipv4_4294967295_1032667358' is in cache**
2022/12/15 02:57:49 [info] 1802#1802: *597 [ACCESS] Return value from crowdsec.access() is : CrowSec bouncer denied request, client: x.x.x.222, server: app.example.com, request: "GET /data/upload/image/20220409/1649486537112450.png HTTP/2.0", host: "app.example.com", referrer: "https://app.example.com/"
2022/12/15 02:57:49 [warn] 1802#1802: *597 [ACCESS] Denied access from crowdsec : CrowSec bouncer denied request, client: x.x.x.222, server: app.example.com, request: "GET /data/upload/image/20220409/1649486537112450.png HTTP/2.0", host: "app.example.com", referrer: "https://app.example.com/"

**Reply by killmasta93** [reply]
hi @andyoulovexy were you able to solve this issue? currently having the same issue which i just realized

**Comment by andyoulovexy** [comment]
Until version 1.5.2, this issue still persists, equivalent to crowdsec still not being used as a plugin

**Reply by killmasta93** [reply]
try to put crowdsec as stream in the global config settings

**Reply by andyoulovexy** [reply]
which docker image tag is stable,I used dev. if set crowdsec stream. any bad ip won't be banned~~

**Reply by andyoulovexy** [reply]
This is compose file :
version: "3.5"

services:
  bunkerweb:
    image: bunkerity/bunkerweb:dev
    ports:
      - 80:8080
      - 443:8443
    labels:
      - "bunkerweb.INSTANCE"
    environment:
      - SERVER_NAME=xxxxxxx
      - MULTISITE=yes
      - USE_VIRUSTOTAL=yes
      - VIRUSTOTAL_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
      - DATABASE_URI=mariadb+pymysql://bunkerweb:xxxxx@bw-db:3306/db # Remember to set a stronger password for the database
      - API_WHITELIST_IP=127.0.0.0/8 10.20.30.0/24
      - DISABLE_DEFAULT_SERVER=yes
      - USE_CLIENT_CACHE=yes
      - USE_GZIP=yes
      - xxxxx_USE_UI=yes
      - xxxxx_USE_REVERSE_PROXY=yes
      - xxxxxx_REVERSE_PROXY_URL=/admin
      - xxxxxx_REVERSE_PROXY_HOST=http://bw-ui:7000
      - xxxxxx_INTERCEPTED_ERROR_CODES=400 404 405 413 429 500 501 502 503 504
    networks:
      - bw-universe
      - bw-services
    logging:
      driver: syslog
      options:
        syslog-address: "udp://10.20.30.254:514"

  bw-scheduler:
    image: bunkerity/bunkerweb-scheduler:dev
    depends_on:
      - bunkerweb
      - bw-docker
    environment:
      - DATABASE_URI=mariadb+pymysql://bunkerweb:xxxxxx@bw-db:3306/db # Remember to set a stronger password for the database
      - DOCKER_HOST=tcp://bw-docker:2375
    networks:
      - bw-universe
      - bw-docker

  bw-docker:
    image: tecnativa/docker-socket-proxy:nightly
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - CONTAINERS=1
      - LOG_LEVEL=warning
    networks:
      - bw-docker

  bw-ui:
    image: bunkerity/bunkerweb-ui:dev
    depends_on:
      - bw-docker
    environment:
      - DATABASE_URI=mariadb+pymysql://bunkerweb:xxxxx@bw-db:3306/db # Remember to set a stronger password for the database
      - DOCKER_HOST=tcp://bw-docker:2375
      - ADMIN_USERNAME=xxxxx
      - ADMIN_PASSWORD=xxxxxx# Remember to set a stronger password for the xxxxxe user
    networks:
      - bw-universe
      - bw-docker

  crowdsec:
    image: crowdsecurity/crowdsec:v1.5.2
    volumes:
      - cs-data:/var/lib/crowdsec/data
      - ./acquis.yaml:/etc/crowdsec/acquis.yaml
      - bw-logs:/var/log:ro
    environment:
      - BOUNCER_KEY_bunkerweb=xxxxxx
      - COLLECTIONS=crowdsecurity/nginx
    networks:
      - bw-universe

  syslog:
    image: balabit/syslog-ng:4.2.0
    volumes:
      - ./syslog-ng.conf:/etc/syslog-ng/syslog-ng.conf
      - bw-logs:/var/log
    networks:
      bw-universe:
        ipv4_address: 10.20.30.254


  bw-db:
    image: mariadb:10.10
    environment:
      - MYSQL_RANDOM_ROOT_PASSWORD=yes
      - MYSQL_DATABASE=db
      - MYSQL_USER=bunkerweb
      - MYSQL_PASSWORD=xxxxxxxx # Remember to set a stronger password for the database
    volumes:
      - bw-data:/var/lib/mysql
    networks:
      - bw-docker

volumes:
  bw-data:
  cs-data:
  bw-logs:

networks:
  bw-universe:
    name: bw-universe
    ipam:
      driver: default
      config:
        - subnet: 10.20.30.0/24
  bw-services:
    name: bw-services
  bw-docker:
    name: bw-docker

**Reply by killmasta93** [reply]
currently using the 1.5.2 and its working fine with stream and does block

**Reply by andyoulovexy** [reply]
just stream?why does't work in live mode?

**Reply by killmasta93** [reply]
i have no idea, but it seems to work like that

**Reply by andyoulovexy** [reply]
Does Bunkerweb1.5.2 still require clamav and virtuotal API containers !!why clamav and virustotal does't work?do you test it?

**Reply by killmasta93** [reply]
yeah clamav does not work havent try Virus total yet still looking into the clam AV issue

**Reply by andyoulovexy** [reply]
ok. I will wait an stable version.
